static/static.js

```diff
@@ -574,3 +574,6 @@ $("#bithub-events-embed").upcomingEvents({
   apiKey: "AIzaSyBsNpdGbkTsqn1BCSPQrjO9OaMySjK5Sns",
   calendarId: "jupiterjs.com_g27vck36nifbnqrgkctkoanqb4@group.calendar.google.com"
 });
+
+// Set year in footer copyright link
+$('.footer-copyright .end-year').text(new Date().getFullYear());
```

Comment:
The year gets updated when the site is built, right? I don’t think we need to also update it on the client.